src/main/java/com/example/bank/bankTransfer/transfer/history/export/XlsxTransferHistoryGenerator.java

```diff
@@ -1,150 +0,0 @@
-package com.example.bank.bankTransfer.transfer.history.export;
-
-import com.example.bank.bankTransfer.transfer.history.TransferHistory;
-import com.example.bank.bankTransfer.transfer.history.TransferHistoryService;
-import lombok.RequiredArgsConstructor;
-import org.apache.poi.ss.usermodel.*;
-import org.apache.poi.xssf.usermodel.XSSFWorkbook;
-import org.jetbrains.annotations.NotNull;
-import org.springframework.stereotype.Service;
-
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.math.BigDecimal;
-import java.time.LocalDateTime;
-import java.util.List;
-import java.util.stream.IntStream;
-
-@Service
-@RequiredArgsConstructor
-public class XlsxTransferHistoryGenerator {
-    private static final String TRANSFER_HISTORY_SHEET_NAME = "Transfer history";
```

Comment:
czemu ta klasa zniknela i nic poza tym sie nie zmienilo ? :D

Reply:
Właśnie ją usunąłem i układam w głowie odpowiedź. 
Wczoraj po pushu git poinformował mnie o konflikcie, który rozwiązałem, choć byłem bardzo zaskoczony, bo ostatnio przecież wszystko mergowałem. Chodziło o pakiety - "bankTransfer" został usunięty i ta klasa powinna też być usunięta. Niestety git po rozwiązaniu konfliktu zrobił commit: 
 [Merge branch 'develop' into feature/transfer-history](https://github.com/rychcicki/bank-app/pull/13/commits/3f01f334f915f5ad18135f387db87ef16aa6c580)
[3f01f33]
do którego wniosłeś uwagi, ale ta klasa już wyglądała inaczej ostatnio. Dodałem też swój wyjątek wg uwag z poprzedniego PR. Było już po północy, więc nie grzebałem w gicie i nie zauważyłem tego commita ze starą klasą w pakiecie //bankTransfer//
De facto PR dotyczył wcześniejszego commita - tego przed rozwiązaniem konfliktu.
[feat: added unit tests and corrected comments](https://github.com/rychcicki/bank-app/pull/13/commits/068b45abcfd13dffb0035cc8baad91dd5083dde0)
[068b45a]
Oszczędź mi szczegółów mojej niekompetencji :-(